starlingcaptureapi/actions.py

```diff
@@ -83,12 +88,40 @@ def create_proofmode(self, asset_fullpath, jwt_payload):
         tmp_asset_file = _asset_helper.get_tmp_file_fullpath(".jpg")
         tmp_claim_file = _asset_helper.get_tmp_file_fullpath(".json")
 
-        # TODO: Unzip bundle, store asset file, and create dictionary for claim creation.
         meta_proofmode = None
-
+        # Unzip bundle, store asset file, and create dictionary for claim creation.
+        with tempfile.TemporaryDirectory() as tmp_unzip_folder:
+            with zipfile.ZipFile(asset_fullpath, 'r') as zip_ref:
+                zip_ref.extractall(tmp_unzip_folder)
+                for file in os.listdir(tmp_unzip_folder):
```

Comment:
Just to be explicit: this assumes the zip only has one level of files, and we don't need to walk the directory inside the zip, right?

Reply:
Correct. Proofmode uses zip as a container for multiple files, not a filesystem.